{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /register (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fedd002f410>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","children":[{"name":"test_registration","uid":"38f9c899f3591aaa","parentUid":"ab61bfa9a0560312da21712d4b98ddd0","status":"broken","time":{"start":1748440877810,"stop":1748441013025,"duration":135215},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ab61bfa9a0560312da21712d4b98ddd0"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}